### PR TITLE
[ConstitutiveLawsApp] Adding more CL to the interface

### PR DESCRIPTION
**Pull request by AlejandroCornejo**
In this PR I'm adding several CL to the Structural ptype

### CODE REVIEW

**Comment by AlejandroCornejo** [comment]
@jginternational If there is something i need to change just let me know

File: kratos.gid/apps/Structural/xml/ConstitutiveLaws.xml [hunk 1]
```diff
@@ -201,4 +201,134 @@
         </inputs>
         <outputs></outputs>
     </CLaw>
+
+    <!--Damage isotropic-->
+    <CLaw n="SmallStrainIsotropicDamage3DVonMisesVonMises" pn="Small Strain Isotropic damage"  Type="3D" Behaviour="Damage" StrainSize="6" ImplementedInApplication="StructuralMechanicsApplication" help="Isotropic damage for 3d solids" Dimension="3D" LargeDeformation="False" RequiresLocalAxes="False" HybridType="False">
```
Comment (rubenzorrilla) [inline kratos.gid/apps/Structural/xml/ConstitutiveLaws.xml:206]: I'd give few more details in the help.

Comment (AlejandroCornejo) [reply kratos.gid/apps/Structural/xml/ConstitutiveLaws.xml]: done

Comment (rubenzorrilla) [reply kratos.gid/apps/Structural/xml/ConstitutiveLaws.xml]: Maybe a bit picky, but I'd add what are these VonMisesVonMises (I guess that these would be the failure surfaces or something of this sort). This nomenclature might be obvious for you but maybe it's not for the users of the GUI 😅 .

Comment (AlejandroCornejo) [reply kratos.gid/apps/Structural/xml/ConstitutiveLaws.xml]: Ok, you're right. Gonna do it

**Comment by AlejandroCornejo** [comment]
is now ready to go? :)

**Comment by loumalouomega** [comment]
> is now ready to go? :)

yes honey

**Pull request merged**